client/src/site-search/index.scss

```diff
@@ -8,12 +8,16 @@
 .site-search {
   display: block;
 
-  article {
+  .main-content {
     @media (max-width: $screen-md) {
       display: flex;
       flex-direction: column;
     }
 
+    article {
+      display: contents;
+    }
```

Comment:
I don't think it's a good idea to have this `article` here behave differently then other `article`s on MDN.

Reply:
Noted, this basically affects the `main-page-content` class. It only affects padding/margin. But this was the least minimal invasive change.